.pre-commit-config.yaml

```diff
@@ -4,6 +4,9 @@ repos:
     hooks:
       - id: flake8
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812
+    rev: v0.942
     hooks:
       - id: mypy
+        additional_dependencies: [
+          types-requests>=2.25.0,
+        ]
```

Comment:
I thought changing `mypy` to `>0.900` was quite a big change? Surprised it doesn't require more type stub packages... In any case, we should split this out and do a PR for both libraries in sync to avoid incompatible dev environments.

Reply:
Yeh tbh I was surprised it was this easy. I simply bumped to `>0.900` and then only added the type stub packages necessary to run successfully. The full list of stdlib type stub packages is [here](https://github.com/python/typeshed/tree/master/stubs) I think. I'm surpised we only seem to need `type-requests` (and `type-urllib3` which is a dep of `type-requests`. But it seems to run OK!

Good idea separating out, I didn't think about compatible dev requirements. I'll do them both together!

Reply:
Great, thanks. I think when I looked into bumping for `alibi` it seemed more involved with `numpy` stuff etc. but things might have improved since 0.900 dropped.

Reply:
Think you might be right, I started to have a go with `alibi`; the type stubs seemed fine but it did raise some additional `numpy` errors. I'll have another look!